css/styles.css

```diff
@@ -242,6 +243,70 @@ body {
   width: 450px;
 }
 
+/* Fixed Phone */
+.fixed-phone {
+  position: fixed;
+  width: 180px;
+  height: 100px;
+  border: none;
+  border-radius: 50px 0 0 50px;
+  -webkit-border-radius: 50px 0 0 50px;
+  -moz-border-radius: 50px 0 0 50px;
+  background-color: #d9d9d9;
+  bottom: 160px;
+  right: 0;
+}
+.fixed-phone__img {
+  cursor: pointer;
+  position: absolute;
+  z-index: 1;
```

Comment:
here we have to decide if the phone icon will be on top of everything in home-page. now it's behind the header. so maybe we should put the navbar to z-index-3 and the phone icon to z-index-2?

Reply:
I would suggest to leave it as it is